.github/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 3. Enhancement - Changed order of defined exits on Ground Networks - thanks to @Hinshee (Will Hinshaw)
 4. Bug - Corrected Manchester (EGCC) AIR Split Airport Ownership - thanks to @PLM1995 (Peter Mooney)
 5. AIRAC (2411) - Added Blackbushe (EGLK) Hold W - thanks to @rishab-alt
+6. AIRAC (2411) - Updated Odiham frequencies - thanks to @aidenbok203 (Aiden Bok)
```

Comment:
```suggestion
6. AIRAC (2411) - Updated Odiham (EGVO) frequencies - thanks to @aidenbok203 (Aiden Bok)
```
It's usual to include the ICAO code after the AD name in brackets in CL entries, PR titles and Issue titles. Just makes things easier to search :)